src/scripts/main.js

```diff
@@ -1 +1,20 @@
 'use strict';
+
+const mainImg = document.querySelector('#largeImg');
+const listImg = document.querySelector('#thumbs');
+
+listImg.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  const target = e.target;
+
+  if (!target) {
```

Comment:
The check `if (!target)` is unnecessary because `e.target` will always be an element when the event is triggered. You might want to check if `target` is an element you expect, such as an `IMG` or `A` tag.